book/website/community-handbook/community-calls.md

```diff
@@ -61,3 +61,19 @@ We want to make collaborative events like these as useful as possible for as man
 Please [let the team members know](https://github.com/the-turing-way/the-turing-way#get-in-touch) if there are other ways we can make these calls more welcoming and helpful for you.
 
 More templates related to Community Calls are provided in the {ref}`community template collection<ch-templates>`.
+
+### Joining Community Calls in Low-Bandwidth Environments
+
+We understand that accessing online community calls can be challenging for individuals in low-bandwidth environments.
+
+Here are some tips and alternatives for joining our calls:
+
+- **Using Zoom Mobile App:** If you're experiencing low bandwidth on your computer, consider joining the call using the Zoom mobile app on your smartphone. The app may provide better connectivity and audio quality.
+
+- **Calling In:** Zoom allows participants to join calls via phone call, which can be a viable option for those with limited internet access. Check the meeting details for dial-in numbers and access codes.
+
+- **Reducing Video Quality:** If video is not essential for your participation, consider disabling your video feed during the call to conserve bandwidth. You can still actively engage in discussions via audio and text chat.
+
+[Learn more How you can Join our Zoom Meeting Calls](https://support.zoom.com/hc/en/article?id=zm_kb&sysparm_article=KB0060732)
+
+We are committed to ensuring that our community calls are accessible to all members. If you have any specific accessibility needs or suggestions, please reach out to us so we can accommodate them.
```

Comment:
```suggestion
We are committed to ensuring that our community calls are accessible to all members. If you have any specific accessibility needs or suggestions, please reach out to us so we can accommodate them. For more information on our access-related practices as a community, please visit our [Community Handbook](https://the-turing-way.netlify.app/community-handbook/accessibility).
```

Reply:
@aleesteele - this should be a [relative link](https://the-turing-way.netlify.app/community-handbook/style/style-crossref) rather than a direct one. 
I know it's nitpicky but remember [semantic line breaks](https://the-turing-way.netlify.app/community-handbook/style#write-each-sentence-in-a-new-line-line-breaks) too 😉